feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.html

```diff
@@ -0,0 +1,53 @@
+<ng-container *ngIf="orderEntry$ | async as orderEntry">
+  <ng-container *ngIf="isBundleBasedConfigurator(orderEntry)">
+    <ng-container *ngIf="(numberOfLineItems$ | async) > 0">
+      <div class="cx-number-items">
```

Comment:
What if we have only one item?

Reply:
A good question that you could answer yourself. Check out this branch, build it and start the server. Then you could check what happends if you select only one line item.

Reply:
See screenshot: